test/testdata/cosigned/e2e/cip-requires-two-signatures-and-two-attestations.yaml

```diff
@@ -111,20 +111,20 @@ spec:
     type: cue
     data: |
       if len(authorityMatches."keyless-att".attestations) < 2 {
-        keylessAttestationsErr: "error"
-        keylessAttestationsErr: "Did not get both keyless attestations"
+        keylessAttestationsErr: 1
+        keylessAttestationsErrMsg: "Did not get both keyless attestations"
```

Comment:
Haha, this was my super hack to raise an error because I don't understand enough of cue yet :)  So, we only execute this if if there's a failure and because there's a mismatch the policy will fail. So, all of these 'branches' really should be:
if len(...) < expected {
  raiseEvaluationError
}

But, I don't know how to do the raiseEvaluationError, so I just added a statement that will result in an error.